Engine/Graphics/Indoor.cpp

```diff
@@ -928,84 +844,68 @@ bool IndoorLocation::Load(const std::string &filename, int num_days_played,
         j += pSector->uNumMarkers;
```

Comment:
Vector out of range here on loading indoor maps

Reply:
Fixed.